Hardcore.lua

```diff
@@ -53,8 +53,11 @@ Hardcore_Character = {
 	deaths = {},
 	bubble_hearth_incidents = {},
 	played_time_gap_warnings = {},
-	trade_partners = {},
+	trade = {},
 	grief_warning_conditions = GRIEF_WARNING_BOTH_FACTIONS,
+	is_valid_run = false,
```

Comment:
Sneaked this in as well. Tbh I am unsure about this, I see 1000 people freaking out in discord because of this.